.github/workflows/menual-cicd.yml

```
@@ -0,0 +1,64 @@
+name: Menual CI/CD
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)